### DISCUSSION TITLE
(suggestion) add more things like "Aim at teammates" or "Aim at custom NPCS" (the customs npcs are coded without HHH or Meramus)

### DISCUSSION THREAD

**Discussion by jaaajjjjjjjjjjjjjjjjjjjj** [Fedoraware/Fedoraware#990] [Ideas]
im an idiot who cant use a template correctly and puts all my information in the title of my issues.

**Comment by jaaajjjjjjjjjjjjjjjjjjjj** [comment]
This would be grateful

**Comment by ghost** [comment]
this will be added probably, dont put all your info in the title of your issue posts.
